### DISCUSSION TITLE
G20 330e 2019 Plug-In Hybrid Fingerprint

### DISCUSSION THREAD

**Discussion by TheFitzZZ** [bimmerconnected/bimmer_connected#186] [Fingerprint data contributions]
No issues I think, but I wasn't sure if you had hybrid fingerprints :-) 
[vehicles_0.txt](https://github.com/bimmerconnected/bimmer_connected/files/4789044/vehicles_0.txt)
[status_0.txt](https://github.com/bimmerconnected/bimmer_connected/files/4789045/status_0.txt)

**Comment by gerard33** [comment]
Thanks! The more data the better 😀
